src/20240320_AaveV3Ethereum_OsETHOnboarding/AaveV3Ethereum_OsETHOnboarding_20240320.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Ethereum, AaveV3EthereumEModes} from 'aave-address-book/AaveV3Ethereum.sol';
+import {AaveV3PayloadEthereum} from 'aave-helpers/v3-config-engine/AaveV3PayloadEthereum.sol';
+import {EngineFlags} from 'aave-helpers/v3-config-engine/EngineFlags.sol';
+import {IAaveV3ConfigEngine} from 'aave-helpers/v3-config-engine/IAaveV3ConfigEngine.sol';
+import {IV3RateStrategyFactory} from 'aave-helpers/v3-config-engine/IV3RateStrategyFactory.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+/**
+ * @title osETH Onboarding
+ * @author Aave Chan Initiative
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x3dc8b06441d0f1dcd6f4a53d06d62e9bb1ac87ced19020d9c735854bbf68b835
+ * - Discussion: https://governance.aave.com/t/arfc-onboard-oseth-to-aave-v3-on-ethereum/16913
+ */
+contract AaveV3Ethereum_OsETHOnboarding_20240320 is AaveV3PayloadEthereum {
+  using SafeERC20 for IERC20;
+
+  address public constant osETH = 0xf1C9acDc66974dFB6dEcB12aA385b9cD01190E38;
+  uint256 public constant osETH_SEED_AMOUNT = 0.01 ether;
+
+  function _postExecute() internal override {
+    IERC20(osETH).forceApprove(address(AaveV3Ethereum.POOL), osETH_SEED_AMOUNT);
+    AaveV3Ethereum.POOL.supply(osETH, osETH_SEED_AMOUNT, address(AaveV3Ethereum.COLLECTOR), 0);
+  }
+
+  function newListings() public pure override returns (IAaveV3ConfigEngine.Listing[] memory) {
+    IAaveV3ConfigEngine.Listing[] memory listings = new IAaveV3ConfigEngine.Listing[](1);
+
+    listings[0] = IAaveV3ConfigEngine.Listing({
+      asset: osETH,
+      assetSymbol: 'osETH',
+      priceFeed: 0x0A2AF898cEc35197e6944D9E0F525C2626393442,
+      eModeCategory: AaveV3EthereumEModes.NONE,
```

Comment:
https://governance.aave.com/t/arfc-onboard-oseth-to-aave-v3-on-ethereum/16913/8 why no eMode (as recommended by risk providers?)